cds_ils/importer/cli.py

```diff
@@ -0,0 +1,23 @@
+import click
+from flask.cli import with_appcontext
+
+from cds_ils.importer.importer import import_from_xml
+
+
+@click.group()
+def importer():
+    """CDS-ILS importer commands."""
+
+
+@importer.command()
+@click.argument("sources", type=click.File("r"), nargs=-1)
+@click.option(
+    "--source-type",
+    "-t",
+    type=click.Choice(["json", "marcxml"]),
+    default="marcxml",
+    help="Import from JSON, MARCXML or CDS-Migrator-Kit's _records.json file.",
```

Comment:
this is great! Maybe someone can help and create a very simple UI to upload the file and report results.